tinytuya/wizard.py

```diff
@@ -220,13 +226,14 @@ def wizard(color=True, retries=None, forcescan=False, nocloud=False, quicklist=F
 
     # Display device list
     print("\n\n" + bold + "Device Listing\n" + dim)
-    output = json.dumps(tuyadevices, indent=4)  # sort_keys=True)
-    print(output)
+    print( json.dumps(tuyadevices[:15], indent=4) )
+    if len(tuyadevices) > 15:
+        print("%s(%d more devices hidden)" % (normal, (len(tuyadevices) - 15)))
```

Comment:
Hum... interesting. I'm not sure we want to truncate the list. It may be better to make the JSON output optional (via command line?) and just notify the user that the device files is written.

Reply:
I'm not sure why we need to display all off them.  A few are nice to verify the download worked, but with the addition of device mappings they're absolutely flooding the screen (I'm up to 130 devices and every time I ran the wizard it was dumping all 14,793 lines of devices.json to my screen).  Perhaps add another prompt?

Reply:
I added a prompt.  If you have <= 15 devices it asks:
> 3 devices downloaded, display? (Y/n):

and more than that gets:
> 130 devices downloaded, display? ([Y]es/[n]o/[s]ome):

As 15 was still flooding my screen, if you answer "s" it then displays just the first 10.